web-jehlomat/src/Components/Navigation/TitleBarStyles.tsx

```diff
@@ -32,7 +32,7 @@ export const NavIcon = styled.div`
 `;
 
 interface IContent {
-    isCentered: boolean;
+    isCentered: boolean | undefined;
```

Comment:
```suggestion
    isCentered?: boolean;
```

Reply:
Toto neni v novem typescriptu to same, pls read documentation for typesceipt

Reply:
Toto v ramci noveho typescriptu neni to same, ale tady nevim jestli je to spravne viz. https://kate-dev.medium.com/when-to-use-typescript-optional-property-how-is-it-different-from-declaring-property-as-undefined-2319a0ee1f07
Optional tzn v objektu value neni, pokud je undefined musi byt value: undefined

Reply:
Jasně že to není to samé, vycházel jsem z toho, že jde o boolean, tedy přišlo mi že záměrem spíš bylo mít optional prop než třístavový boolean, ale abych se přiznal, tak jsem dále nezkoumal použití, takže pokud jsem se unáhlil, tak mě prosím @djdoma oprav. A pokud jo, tak bych zvážil přejmenování na něco jiného než `isCentered`, jde o to, že prefix `is` v názvu se většinou používá právě na boolean typy.